modules/options/src/main/scala/scala/build/interactive/Interactive.scala

```diff
@@ -36,7 +36,11 @@ object Interactive {
       def msg: String
       def action: Option[V]
       final def run: Option[V] =
-        if (interactiveInputsOpt.nonEmpty || coursier.paths.Util.useAnsiOutput())
+        if (
+          interactiveInputsOpt.nonEmpty ||
+          coursier.paths.Util.useAnsiOutput() ||
+          System.getenv("SCALA_CLI_INTERACTIVE_TEST") != null
```

Comment:
```suggestion
          System.getenv("SCALA_CLI_INTERACTIVe") != null
```

This flag could be useful not only in the tests.